src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts

```diff
@@ -387,6 +388,7 @@ export class DebugEditorContribution implements IDebugEditorContribution {
 	private showEditorHover(position: Position, focus: boolean) {
 		const hoverController = this.editor.getContribution<ModesHoverController>(ModesHoverController.ID);
 		const range = new Range(position.lineNumber, position.column, position.lineNumber, position.column);
+		this.enableEditorHover();
 		hoverController?.showContentHover(range, HoverStartMode.Immediate, HoverStartSource.Mouse, focus);
```

Comment:
Can it disable the hover right after showing it here?

Reply:
Then we would have the initial problem where mouse move will lose track of editor hover